src/dotnet/HelpException.cs

```diff
@@ -6,7 +6,6 @@ namespace Microsoft.DotNet.Cli
     /// 
     /// <summary>Allows control flow to be interrupted in order to display help in the console.</summary>
     ///
-    [Obsolete("This is intended to facilitate refactoring during parser replacement and should not be used after that work is done.")]
```

Comment:
I feel like this is not what the original author intended....

@jonsequitur - thoughts?

Reply:
File a bug. Obsolete as todo is just silly. It clearly had no impact.

Reply:
I got really upset trying to find the real issues among a sea of warnings. From now on, a warning in our build is an error.

Reply:
https://msdn.microsoft.com/en-us/library/system.net.websockets.websocket.isapplicationtargeting45(v=vs.110).aspx

For years, the obsolete message was something like "this is temporary until we get a new API from CLR team". It's permanent, you can call it in netstandard 2.0. 😡